src/tokens/TokenDetails.tsx

```diff
@@ -92,14 +96,25 @@ export default function TokenDetailsScreen({ route }: Props) {
           errorFallback={(localCurrencySymbol ?? '$').concat(' --')}
         />
         {!token.isStableCoin && <PriceInfo token={token} />}
-        {token.isNative && token.symbol === 'CELO' && (
-          <CeloGoldHistoryChart
-            color={Colors.black}
-            containerStyle={styles.chartContainer}
-            chartPadding={Spacing.Thick24}
-            testID="TokenDetails/Chart"
-          />
-        )}
+        {usePriceHistoryFromBlockchainApi
+          ? !token.isStableCoin && (
```

Comment:
do we want to show the chart for all non stable coin tokens? I think the original [design](https://www.figma.com/file/OE6RFzqXAMqU9ygHtNYyof/Assets-Page-(Multichain)?node-id=2099%3A17366&mode=dev) only included charts for native tokens. I don't think it hurts showing the chart for everything except stables, but would be good to get product confirmation.

Reply:
Handling tokens with a price < $0.01 does at a bit of complexity, but it looks like we could adjust `getLocalCurrencyDisplayValue` to take an optional param to resolve this also. We just need to derive what number of significant digits we need based on the high and low prices.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-01-03 at 18 17 56](https://github.com/valora-inc/wallet/assets/26950305/a2f9e22d-4b33-4029-b1bd-6ddd2c14e360)